video_mapper.go

```diff
@@ -420,7 +425,16 @@ func getByline(brightcoveVideo map[string]interface{}, publishReference string)
 	return byline
 }
 
-func get(key string, brightcoveVideo map[string]interface{}) (val string, err error) {
+func getFirstPublishedDate(video map[string]interface{}) (result string) {
+	result, err := get("published_at", video)
+	if err != nil {
+		warnLogger.Printf("No valid value could be found for firstPublishedDate: [%v]", err)
+	}
+
+	return result
```

Comment:
In short functions with named results types, you could simply use a simple return.